<launch>
    <node name="image_source_dummy" pkg="image_source" type="image_source_dummy.py">
        <param name="width" value="1920" />
        <param name="height" value="1080" />
        <param name="frame_rate" value="10"/>
    </node>
</launch>